.env.example

```
@@ -4,6 +4,7 @@ VITE_POLYGON_RPC_URL=https://rpc-mumbai.maticvigil.com/
 VITE_POLYGON_INDEXER_URL=
 VITE_POLYGON_MIGRATION_START_TIME=
 
+VITE_MIGRATION_LEARN_MORE_URL=https://arkscic.com/blog/ark-is-moving-to-polygon
 VITE_MIGRATION_GUIDE_URL=https://arkvault.io/docs
```

```suggestion
VITE_MIGRATION_GUIDE_URL=https://ark.dev/docs/core/migration/devnet
```

to avoid default linking to incorrect places